config/manifests/bases/falcon-helm.clusterserviceversion.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: '[]'
+    capabilities: Basic Install
+  name: falcon-helm.v0.0.0
```

Comment:
Other places use different version names (eg alpha).

Reply:
I am not entirely sure what you are asking here, but I think the `maturity` key is what we want to denote for `alpha` or not.

Reply:
I think keeping the `alpha` in the `maturity` key makes sense vs the name. I want to keep versioning and maturity as is right now because I don't feel the operator part is fully there yet for this helm chart.